app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -439,7 +439,7 @@ def parse_component_status(container_component_status)
         :condition => component_condition.type,
         :status    => component_condition.status,
         :message   => component_condition.message,
-        :error     => component_condition.error
+        :error     => (component_condition.error unless component_condition.error == "nil")
```

Comment:
@dkorn this seems a problem that should be addressed in OpenShift/Kubernetes (not sure if it comes from Kubernetes or if it's something specific that OpenShift introduced, it requires some research). Please open the issue on the relevant project.